docs/src/content/hardhat-runner/docs/getting-started/index.md

```diff
@@ -239,56 +239,84 @@ Compiled 2 Solidity files successfully
 
 ### Deploying your contracts
 
-:::tip
-
-Try [Hardhat Ignition](/ignition) for your deployments! Our new declarative system for deploying smart contracts without getting caught up in execution details.
-
-:::
-
-Next, to deploy the contract we will use a Hardhat script.
+Next, to deploy the contract we will use a Hardhat Ignition module.
 
-Inside the `scripts/` folder you will find a file with the following code:
+Inside the `ignition/modules` folder you will find a file with the following code:
 
 ::::tabsgroup{options="TypeScript,JavaScript"}
 
 :::tab{value="TypeScript"}
 
-<<< @/../packages/hardhat-core/sample-projects/typescript/scripts/deploy.ts
+<<< @/../packages/hardhat-core/sample-projects/typescript/ignition/modules/LockModule.ts
 
 :::
 
 :::tab{value="JavaScript"}
 
-<<< @/../packages/hardhat-core/sample-projects/javascript/scripts/deploy.js
+<<< @/../packages/hardhat-core/sample-projects/javascript/ignition/modules/LockModule.js
 
 :::
 
 ::::
 
-You can run it using `npx hardhat run`:
+You can deploy it using `npx hardhat ignition deploy ./ignition/modules/LockModule.ts`:
```

Comment:
Should this indicate that it running deploy is safe as it is against an in-memory Hardhat node?